includes/class-wc-payments-tasks.php

```diff
@@ -23,6 +23,10 @@ class WC_Payments_Tasks {
 	public static function init() {
 		include_once WCPAY_ABSPATH . 'includes/admin/tasks/class-wc-payments-task-disputes.php';
 
+		if ( ! current_user_can( 'manage_woocommerce' ) ) {
```

Comment:
- `admin_init` does not work
- I've moved the `include_once` inside the `add_task_disputes_need_response` function

I've left the `current_user_can()` before the hook. I'm making the assumption here that this call is performant, either way it will need to run on every request (due to the need for the `init` hook.) The include and the WooPayments checks do not need to be run if the permission checks fails. Likely a micro-optimization, but a performance improvement regardless.

I'm happy to include the check inside the `add_task_disputes_need_response` function if there is good reason.